src/main/resources/db/migration/2001234.sql

```diff
@@ -0,0 +1,3 @@
+# 2.1.194
```

Comment:
```suggestion
# 2.2.3
```

Reply:
Please rename this file to match the current version in `pom.xml`. The current version is `2.2.3`.

Reply:
Done

Reply:
@Shubham-Vishwakarma Can you also rename the file itself?

Reply:
Done